File: Web/src/app/models/Palestrante.ts
import { Rede } from "./Rede";
import { Evento } from "./Evento";

export interface Palestrante {
    id: number; 
    nome: string;
    descricao: string;
    imagemUrl: string;
    telefone: string;
    email: string;
    redes: Rede[]; 
    palestranteEventos: Evento[]; 
}

File: Api/Models/Palestrante.cs
using System.Collections.Generic;

namespace Api.Models
{
    public class Palestrante
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Descricao { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public string ImagemUrl { get; set; }
        public List<Rede> Redes { get; set; }
        public List<PalestranteEvento> PalestrantesEventos { get; set; }
    }
}